src/supported-value.js

```diff
@@ -67,14 +78,21 @@ export default function supportedValue(property, value) {
 
     el.style[property] = value
 
-    // Value is supported with vendor prefix.
-    if (el.style[property] !== '') cache[cacheKey] = value
+    if (el.style[property] === '') {
+      cache[cacheKey] = false
+      return false
+    }
   }
 
-  if (!cache[cacheKey]) cache[cacheKey] = false
-
   // Reset style value.
   el.style[property] = ''
 
+  if (isImportant) {
+    value += ' !important'
+    isImportant = false
```

Comment:
why do you set it to false?

Reply:
Because every run of `supportedValue` function, we need `isImportant` to reset it (https://github.com/cssinjs/css-vendor/blob/08d2f32d671238df0ffd62d5be6f8e2524cca5d6/src/supported-value.js#L53), if `!important` exist in value

Reply:
Omg just realized isImportant is outside of the function scope, why? Are you using it outside of that function? __This is a very dangerous side-effect style coding we should always avoid as far as possible__